docs/content/version-reporting.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Anonymous Version Reporting
+kind: operations
+weight: 50
+---
+
+This document provides details about OPA's anonymous version reporting feature.
+
+## Overview
+
+OPA supports anonymous version reporting wherein OPA would reach out to an external service and share its version 
```

Comment:
Nit: Use active voice here.

Instead of

```
OPA supports anonymous version reporting wherein OPA would reach out to an external service and share its version...
```

```
OPA periodically reports its version to publicly hosted, external service. The version reports only contain the OPA version number (e.g., v0.12.3) and a randomly generated UUID.
```

Reply:
Updated